R/helpers.R

```diff
@@ -144,6 +144,12 @@ do_trace <- function(current, risk, mstart,
 check_y_family <- function(y, family)
     family@check_y(y)
 
+## check function for response matrix as in FDboost
```

Comment:
I do not think that it is possible to solve this by such a generic function, as for a call with `mboostLSS()` and `mboost()` the reponse must be a vector and NOT a matrix, i.e., for a matrix-response in this case the error is wanted. 

To make the code in mboostLSS_fit() more readable, I will change `check_y_family()` and give it a new argument `allow_matix = FALSE`, which is set to `TRUE` for FDboost.